pkgs/by-name/co/cosmic-session/package.nix

```
@@ -8,19 +8,20 @@
   stdenv,
   xdg-desktop-portal-cosmic,
 }:
-rustPlatform.buildRustPackage rec {
+
+rustPlatform.buildRustPackage (oldAttrs: {
   pname = "cosmic-session";
-  version = "1.0.0-alpha.2";
+  version = "1.0.0-alpha.6";
 
   src = fetchFromGitHub {
     owner = "pop-os";
-    repo = "cosmic-session";
-    rev = "epoch-${version}";
-    hash = "sha256-rkzcu5lXKVQ5RfilcKQjTzeKZv+FpqrtARZgGGlYKK4=";
+    repo = oldAttrs.pname;
```

```suggestion
    repo = "cosmic-session";
```
Avoid `repo = pname`